modules/introduction/pages/editions.adoc

```diff
@@ -7,39 +7,33 @@
 Each edition offers different features and levels of support.
 
 Enterprise Edition::
```

Comment:
It's not in the issue, but I would make this a heading rather than a definition list, mainly so that users can navigate around the page more easily.

```suggestion
== Enterprise Edition
```